WordPress/Classes/ViewRelated/Reader/ReaderDetailCoordinator.swift

```diff
@@ -124,6 +145,26 @@ class ReaderDetailCoordinator {
         })
     }
 
+    /// Requests a ReaderPost from the service and updates the View.
+    ///
+    /// Use this method to fetch a ReaderPost from a URL.
+    /// - Parameter url: a post URL
+    private func fetch(_ url: URL) {
+        service.fetchPost(at: postURL,
```

Comment:
shouldn't this be ` service.fetchPost(at: url,` ?

Reply:
Fixed in 285e1372d90ff096426537c6470256d72147cc1a